sim/pokemon.ts

```diff
@@ -602,33 +602,21 @@ export class Pokemon {
 	}
 
 	/** Get targets for Dragon Darts */
-	getSmartTargets(target: Pokemon, move: Move) {
+	getSmartTargets(target: Pokemon, move: ActiveMove) {
+		this.battle.attrLastMove('[still]');
```

Comment:
Why? I prefer to special-case it for nicer-looking logs.

Reply:
Also `attrLastMove` is very not-functional side-effect-wazoo, so I'd prefer to avoid it where possible.

Reply:
Actually it's the intended target being excluded and both hits going to the other target that was the special case I was trying to solve here.

Reply:
Ohh, that makes sense.

`retargetLastMove` _does_ exist. I think we should just always call it on the first hit.